mne/io/kit/kit.py

```diff
@@ -720,9 +720,12 @@ def get_kit_info(rawfile, allow_unknown_format, standardize_names=None,
                     hsp.append(rr)
 
             # nasion, lpa, rpa, HPI in native space
-            elp = [dig.pop(key) for key in (
-                'fidnz', 'fidt9', 'fidt10',
-                'hpi_1', 'hpi_2', 'hpi_3', 'hpi_4')]
+            elp = []
+            for key in (
+                    'fidnz', 'fidt9', 'fidt10',
+                    'hpi_1', 'hpi_2', 'hpi_3', 'hpi_4'):
+                if key in dig:
+                    elp.append(dig.pop(key))
             if 'hpi_5' in dig and dig['hpi_5'].any():
                 elp.append(dig.pop('hpi_5'))
```

Comment:
it seems this if just below is exactly to catch the issue of missing hpi_5. Maybe we can use the same logic for all hpi's ?

Reply:
Yes I think that's what it's doing, but not sure why it was checking specifically for hpi_5 and not the others.

Also, based on the additional condition `dig['hpi_5'].any()`, it seems like [0, 0, 0] is considered as invalid coordinates. Again not sure if this is something specific to hpi_5 or should be checked for all marker coils. I don't think we should treat [0, 0, 0] as invalid in general though.

Reply:
> Yes I think that's what it's doing, but not sure why it was checking specifically for hpi_5 and not the others.

I added this because there was one dataset I think was from @paulsowman that had the first four only. But I suppose it makes sense that you might be missing any one (or up to two?) of the five.

Reply:
It looks like this special case is no longer needed. All `hpi_*` should be treated the same way.

While unifying it, we can try getting rid of the `.any` and see if it breaks anything. Maybe `np.isfinite(dig['hpi_5']).all()` is a better check.

Reply:
Agreed and implemented. Doesn't seem to break anything.